app/locale/en_US/Mage_Contacts.csv

```diff
@@ -10,9 +10,10 @@
 "Email Sender","Email Sender"
 "Email Template","Email Template"
 "Enable Contact Us","Enable Contact Us"
+"Invalid Form Key. Please submit your request again.","Invalid Form Key. Please submit your request again."
 "Name","Name"
 "Send Emails To","Send Emails To"
 "Submit","Submit"
 "Telephone","Telephone"
-"Unable to submit your request. Please, try again later","Unable to submit your request. Please, try again later"
+"Unable to submit your request. Please, try again later","Unable to submit your request. Please, try again later."
```

Comment:
The comma can be kept in the first sentence, as I see that it has been used since Magento, but I would remove it from the second sentence. Although it seems like a minor issue, I try to maintain a standard way of elaborating the phrases, which unfortunately did not exist in Magento. This is mainly due to the fact that the developers were not native English speakers.